rst_files/getting_started.rst

```diff
@@ -310,14 +285,6 @@ contents of the current working directory
 These shell commands are handled by your default system shell and hence are platform specific
 
 
-Package Manager
-^^^^^^^^^^^^^^^^
-
-You can enter the package manager by prepending a ``]``
-
-For example, ``] st`` will give the status of installed packages in the current environment
```

Comment:
I don't like ripping this out.  People need to know about the package manager, and there is nowhere else they would learn it.

Reply:
Earlier in the lecture we've basically covered all this info when we initially add the Instantiate package

Reply:
That was a one-time setup.